csi/moac/test/volumes_test.js

```diff
@@ -302,13 +332,18 @@ module.exports = function () {
         limitBytes: 50,
         protocol: 'nbd'
       });
+      await waitUntil(() => volume.state === 'faulted', 'faulted volume');
       sinon.assert.notCalled(stub2);
       sinon.assert.notCalled(stub3);
-      sinon.assert.notCalled(stub1);
+      sinon.assert.calledOnce(stub1);
+      sinon.assert.calledWithMatch(stub1.firstCall, 'createNexus', {
+        uuid: UUID,
+        size: 10,
+        children: [`bdev:///${UUID}`]
+      });
       expect(Object.keys(volume.replicas)).to.have.lengthOf(1);
       expect(Object.values(volume.replicas)[0]).to.equal(replica);
-      expect(volume.state).to.equal('healthy');
-      expect(volEvents).to.have.lengthOf(3);
+      expect(volEvents).to.have.lengthOf(4);
       expect(volEvents[0].eventType).to.equal('new');
       expect(volEvents[1].eventType).to.equal('mod');
       expect(volEvents[2].eventType).to.equal('mod');
```

Comment:
Given that we have an additional event, should we be adding another check for the additional event i.e. volEvents[3]?

Reply:
yup, you are right 👍